### PR TITLE
update kernels-community/aiter-rope version

### PR DESCRIPTION
**Pull request by Abdennacer-Badaoui**
As per title. 
Related to : https://github.com/huggingface/kernels-community/pull/974

### CODE REVIEW

File: src/transformers/integrations/hub_kernels.py
Comment (vasqu) [inline src/transformers/integrations/hub_kernels.py:1]: Can we add a small test that requires rocm so we don't encounter this again? Maybe a kernels test with a small dummy model 🫡

**Review by vasqu — APPROVED** [review]
LGTM, would just add a small test so we might catch this earlier

File: src/transformers/integrations/hub_kernels.py
Comment (Abdennacer-Badaoui) [reply src/transformers/integrations/hub_kernels.py:1]: Happy to add it. Since the breakage was a signature mismatch (`aiter-rope` still carried the old `apply_rotary_transformers(..., position_ids)` after the kernel dropped it and added `can_torch_compile=True`), I'd go with a functional test gated behind `require_rocm `+ `@slow`: load a tiny dummy model with `use_kernels=True` on ROCm and compare logits against the non-kernelized baseline. That actually dispatches through` apply_rotary_transformers`, so a stale signature/version blows up; exactly what slipped through here. A pure signature-comparison test would be more targeted but fragile (compiled kernel funcs often expose only (*args, **kwargs) to `inspect.signature`, so it can silently pass), and it still needs ROCm to load the kernel anyway; so the forward test is strictly more robust for the same hardware requirement. Sound good?

Comment (vasqu) [reply src/transformers/integrations/hub_kernels.py:1]: Yea sure, sounds good!

File: tests/kernels/test_kernels.py [hunk 1]
```diff
@@ -690,3 +691,48 @@ def test_single_device_mapping_still_works(self):
 
         result_mapping = kernel_config.kernel_mapping
         self.assertIn("RMSNorm", result_mapping, "RMSNorm should be in mapping")
+
+
+@require_kernels
+@require_rocm
+@slow
+class TestRocmRotaryKernel(TestCasePlus):
```
Comment (vasqu) [inline tests/kernels/test_kernels.py]: Imo we can add it under https://github.com/Abdennacer-Badaoui/transformers/blob/65d46ca47ef29547eb82783a9bc15ceebdb4636a/tests/kernels/test_kernels.py#L57

no need for a full new set here (just needs that on additional decorator then)

Comment (Abdennacer-Badaoui) [reply tests/kernels/test_kernels.py]: ok let me move it there then

**Comment by vasqu** [comment]
Perfect, merging :saluting_face: thanks a lot for pushing on the AMD side, super nice
